lib/msal-node/docs/caching.md

```diff
@@ -59,7 +59,8 @@ MSAL Node's cache schema is compatible with other MSALs. By default, MSAL's cach
 ```
 
 ## Cache in memory
-If a user chooses not to persist cache, the `TokenCache` interface is still available to access the tokens cached in memory. The life time of in memory cache is the same as MSAL instance. If the MSAL instance  restarts, the cache is erased when the process lifecycle finishes. We recommend persisting the cache with encryption for all real life applications both for security and desired cache longevity.
```

Comment:
For the above: 

1. remove all detailsa about cache schema - this is internal implementation detail.
2. Do not mention memory cache only for desktop apps.

Reply:
done and done